spec/models/post_spec.rb

```diff
@@ -3,4 +3,26 @@
 RSpec.describe Post, type: :model do
   # En base al modelo de test propuesto para el modelo User,
   # implementar los tests para el modelo Post.
+  describe 'Factory' do
+    it 'has a valid factory' do
+      build(:post).should be_valid
```

Comment:
Pese a que el test pasa y está bien, podemos usar un `build_stubbed` para no persistir al `Post` en la base de datos (para mejorar la performance). El `build` es mejor en este sentido que el `create` y lo veo muy bien, pero aún así persiste aquellas asociaciones que tenga definido el _factory_. Es sólo un detalle, pero está bueno conocerlo.

```ruby
build_stubbed(:post)
```

---

<img width="1269" alt="Captura de pantalla 2019-05-27 a la(s) 21 24 32" src="https://user-images.githubusercontent.com/8835404/58443051-d9a6a300-80c5-11e9-9d0a-5fffaea15a87.png">
<img width="492" alt="Captura de pantalla 2019-05-27 a la(s) 21 24 38" src="https://user-images.githubusercontent.com/8835404/58443052-d9a6a300-80c5-11e9-8ac3-5d9aa1886020.png">

Más allá de eso, este test debería tirarte un _warning_ porque estás utilizando un `should` en lugar de un `expect` (no es lo mismo que los `should` que usás después). Con el cambio quedaría de la siguiente forma:

```ruby
it 'has a valid factory' do
  expect(build_stubbed(:user)).to be_valid
end
```